Iterable/iterable-web-sdk
# Embedded message button click

## JIRA Ticket

* [MOB-7510](https://iterable.atlassian.net/browse/MOB-7510)

## Description

> We've successfully addressed the issue where the secondary button click was not being executed.

## Test Steps

> Begin by launching the sample app, proceed to select the 'embedded Msg' option. Enter your userId/email in the designated field and click the submit button. Subsequently, choose the desired option for testing and verify the button click.

[MOB-7510]: https://iterable.atlassian.net/browse/MOB-7510?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
